src/Uno.UI.Toolkit/UIElementExtensions.cs

```diff
@@ -27,12 +27,12 @@ public static class UIElementExtensions
 	{
 		#region Elevation
 
-		public static void SetElevation(this UIElement element, double elevation)
+		public static void SetElevation(UIElement element, double elevation)
```

Comment:
This is unfortunately a breaking change. @carldebilly do you know if the extension method syntax is actually used somewhere ?

Reply:
Oh, ok, sorry. Then how about creating a different class with these same extension methods that would set the attached property, while making the methods non-extension methods? It wouldn't break the callers who use it as an extension method

Reply:
@artemious7 Is it still the case that this causes issues for the designer?

Reply:
We definitely need to re-evaluate whether this breaking change is needed. See https://github.com/unoplatform/uno/issues/5513#issuecomment-1532694800